profile.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header class="header">
+      <h1 class="header__headline">Quiz App</h1>
+    </header>
+
+    <main>
+      <section class="card card__profile">
+        <svg
+          role="img"
+          aria-label="Profil icon"
+          xmlns="http://www.w3.org/2000/svg"
+          width="40"
+          height="2rem"
+          viewBox="0 0 48 48"
+        >
+          <path
+            d="M24 4c-11.05 0-20 8.95-20 20s8.95 20 20 20 20-8.95 20-20-8.95-20-20-20zm0 6c3.31 0 6 2.69 6 6 0 3.32-2.69 6-6 6s-6-2.68-6-6c0-3.31 2.69-6 6-6zm0 28.4c-5.01 0-9.41-2.56-12-6.44.05-3.97 8.01-6.16 12-6.16s11.94 2.19 12 6.16c-2.59 3.88-6.99 6.44-12 6.44z"
+          />
+          <path d="M0 0h48v48h-48z" fill="none" />
+        </svg>
+        <span>Jane Doe</span>
+        <p>
+          ______________<br />
+          ______________<br />
+          ______________
+        </p>
+      </section>
+
+      <section>
+        <button class="button__profile">
```

Comment:
You used a `button` element, but this part are only stats with no (later) user interaction or? Then a `p` element or event a list may be better.